src/Containers/bookslist.js

```diff
@@ -4,47 +4,73 @@ import { connect } from 'react-redux';
 import Msgtitle from '../Components/msgtitle';
 import Booksform from './booksform';
 import Book from './book';
+import Categoriesform from './categoriesform';
 
 function Bookslist(props) {
-  const { bookstore } = props;
+  const {
+    bookstore, filtered, REMOVE_BOOK, UPDATE_BOOK,
```

Comment:
Why these two functions `REMOVE_BOOK`,`UPDATE_BOOK` are all upper case :thinking:

Reply:
sometimes happens that i do no brain things ... like just write again same doc references xd.